4_turb_abl_fsi/template_files/static_box.txt

```diff
@@ -0,0 +1,9 @@
+4
```

Comment:
for a 20B grid - 

4
1
500.0 500.0 0.0 4500.0 4500.0 300.0
1
500.0 500.0 0.0 4500.0 4500.0 300.0
1
500.0 500.0 0.0 4500.0 4500.0 300.0
1
500.0 500.0 0.0 4500.0 4500.0 300.0